README.md

```diff
@@ -1 +1,66 @@
-# To-Do-list
\ No newline at end of file
+# Awesome-books
+This project is part of the 2nd Module 2 Block of Microverse curriculum. The main objective is to use webpack to bundle JavaScript, learn how to use proper ES6 syntax, use ES6 modules to write modular JavaScript, and learn how to use JavaScript objects and arrays and dynamically modify the DOM and add basic events. This will follow professional guidelines in responsive design.
+
+This project will be a basic website that allows users to add, modify and remove a list of To Do tasks.
+
+# Built With
+
+    .HTML
+    .CSS
+    .HTML & CSS Linters
+
+
+# Getting Started
+
+
+Please get your browser update. To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+
+navigate to your projects folder
+
+run git clone (https://github.com/VSeray/Portfolio) in your terminals
+
+run cd microverse_portfolio
+
+start index.html
+
+
+# Authors
+
+
+👤 Author
+
+- GitHub: [github](https://github.com/VSeray)
+
+- Twitter: [twitter](https://twitter.com/home)
+
+- LinkedIn: [Linkdin](https://www.linkedin.com/in/vana-seraydarian-936687191/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BNyso4dw6Tz6UBL%2Fqkjvtvw%3D%3D)
+
+👤 Author
+
+- GitHub: [@Anny85-code](https://github.com/Anny85-code)
+
+- Twitter: @Annyudo8
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aniekan-udo-665b65213/)
```

Comment:
- [ ] I believe this project is solo. Kindly remove this second author for proper documentation.